index.html

```diff
@@ -6,6 +6,6 @@
     <title>Nep huiswerkopdracht</title>
 </head>
 <body>
-
+dit is de verandering
```

Comment:
`Lekker duidelijk... Wat is dan de verandering?

Reply:
hwufhwaeubgfjf